### DISCUSSION TITLE
New task cannot be entered

### DISCUSSION THREAD

**Discussion by Verdifan** [jamiebrynes7/obsidian-todoist-plugin#256] [Q&A] [answered]
When I try to enter a new task in Todoist with the following code from Obsidian, I get to the Todoist interface, but not to the small input window. I have to start it manually by pressing PLUS. Can someone please check the code?
Thx VF

`
+++ button
name Create New Todoist Task
type link
class obsidian-button
action https://app.todoist.com/app://addtask
+++

`

**Comment by Verdifan** [comment]
Okay. This is working with iOS

`
button 
name Create New Todoist Taks
type link
class obsidian-button
action todoist://addtask

`
So works fine with iOS  but not with Windows 10. I have installed the Todoist Windows APP, but unlike iOS, it starts but no task is triggered there. 
VF

**Reply by jamiebrynes7** [reply, marked as answer]
Hi @Verdifan, this block isn't part of this plugin - I'm not sure exactly where its from. Sorry!
